taxonomies/ingredients.txt

```diff
@@ -47097,46 +47122,40 @@ carbon_footprint_fr_foodges_ingredient:fr:Épices
 carbon_footprint_fr_foodges_value:fr:2.2
 # 903 products in 4 languages @2018-10-08
 
+en:herb
+de:Gewürz
+fr:herbe, herbes, plantes aromatiques, plante aromatique, fines herbes, fine herbe, mélange de plantes aromatiques, plantes aromatiques
+nl:Kruiden
 
-fr:épices et plantes aromatique, épice et plante aromatique, épice et plantes aromatiques, épices et plantes aromatiques, épices et plante aromatique
-
-fr:épice et aromate, épices et aromates, épices et aromate, épice et aromates
+<en:herb
+<en:spice
+en:herbs and spices
+fr:épices et plantes aromatique, épice et plante aromatique, épice et plantes aromatiques, épices et plantes aromatiques, épices et plante aromatique, plantes aromatiques et épices, extraits d'épices et d'herbes, herbe et épice, herbes et épices, épices et herbes, épices et herbes aromatiques, épice et aromate, épices et aromates, épices et aromate, épice et aromates, épices et arômes, aromates et épices
 
+<en:herbs and spices
 fr:extraits d'épices et d'aromates
 
 <en:spice
-fr:plantes aromatiques et épices
-
+en:spice extracts
```

Comment:
we should remove the <en:spice parent

Reply:
The idea is to have to parents spice and herb.